lib/db/schema/patch-049-048.sql

```diff
@@ -0,0 +1,6 @@
+-- DROP PROCEDURE `createAccount_7`;
+-- DROP PROCEDURE `accountEmails_2`;
+-- DROP PROCEDURE `forgotPasswordVerified_6`;
+-- DROP PROCEDURE `verifyEmail_5`;
+
+-- UPDATE dbMetadata SET value = '48' WHERE name = 'schema-patch-level';
```

Comment:
Should we also add a migration to copy the existing state from the `accounts` table into the `emails` table?  We probably wouldn't use it in production and would instead manage that copying-over by hand, but doing it in a migration here would put e.g. dev environments and self-hosters in the right state automatically.